### PR TITLE
Create TestFile1

### PR DESCRIPTION
**Pull request by ghost**
A new file is created into a new branch

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Did a self review.

File: TestFile1 [hunk 1]
```diff
@@ -0,0 +1 @@
+This is a simple test file to test a commit, branch, pull and merge.
```
Comment (ykrishk) [inline TestFile1:1]:
```suggestion
This is a simple test file to test a commit, branch, pull and merge.
``` 
This is a good first line. Keep it.

**Review by ghost — COMMENTED** [review]
Added another review comment.

**Comment by ykrishk** [comment]
Thanks for the input. Changes done.

**Comment by ykrishk** [comment]
Reopening pull request

**Comment by ykrishk** [comment]
Ready to pull.

**Review by ghost — COMMENTED** [review]
Reviewed

**Pull request merged**